### ISSUE TITLE
False positive :missing-docstring on `cljs.core/deftype` and `cljs.core/defrecord`

### ISSUE THREAD

**Issue by serioga** [clj-kondo/clj-kondo#1801]
Similar to https://github.com/clj-kondo/clj-kondo/issues/800 but in ClojureScript.

```clj_kondo.cljs
(ns user.clj-kondo
  {:clj-kondo/config {:linters {:missing-docstring {:level :warning}}}})

(deftype MyType [x])

(defrecord MyRecord [x])
```

```
test\user\clj_kondo.cljs:5:1: warning: Missing docstring.
test\user\clj_kondo.cljs:7:1: warning: Missing docstring.
```
